data/css-font-4.json

```diff
@@ -0,0 +1,3 @@
+{
+  "TR": "https://www.w3.org/TR/css-fonts-4/"
```

Comment:
Could you update the name of the file so that it matches the shortname?
In other words, "css-fonts-4.json" instead of "css-font-4.json"

Reply:
Thanks @tidoust, done